### PR TITLE
[New Rule] Potential Credential Access via LSASS Memory Dump

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/1532

### CODE REVIEW

File: rules/windows/credential_access_suspicious_lsass_access_memdump.toml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.code == 10 and
+  winlog.event_data.TargetImage :"?:\\WINDOWS\\system32\\lsass.exe" and
+
+  /* DLLs exporting MiniDumpWriteDump API to create an lsass mdmp */
+  winlog.event_data.CallTrace : ("*dbhelp*", "*dbgcore*") and
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/credential_access_suspicious_lsass_access_memdump.toml:30]: Ref: https://www.pinvoke.net/default.aspx/dbghelp/MiniDumpWriteDump.html

File: rules/windows/credential_access_suspicious_lsass_access_memdump.toml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2021/10/07"
+maturity = "production"
+updated_date = "2021/10/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious access to LSASS handle from a call trace pointing to DBGHelp.dll or DBGCore.dll which both export
```
Comment (brokensound77) [inline rules/windows/credential_access_suspicious_lsass_access_memdump.toml:9]:
```suggestion
Identifies suspicious access to LSASS handle from a call trace pointing to DBGHelp.dll or DBGCore.dll, which both export
```

File: rules/windows/credential_access_suspicious_lsass_access_memdump.toml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+risk_score = 73
+rule_id = "9960432d-9b26-409f-972b-839a959e79e2"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.code == "10" and
+  winlog.event_data.TargetImage :"?:\\WINDOWS\\system32\\lsass.exe" and
```
Comment (brokensound77) [inline rules/windows/credential_access_suspicious_lsass_access_memdump.toml:27]:
```suggestion
  winlog.event_data.TargetImage : "?:\\WINDOWS\\system32\\lsass.exe" and
```

File: rules/windows/credential_access_suspicious_lsass_access_memdump.toml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+ /* DLLs exporting MiniDumpWriteDump API to create an lsass mdmp*/
+  (winlog.event_data.CallTrace : "*dbghelp*" or winlog.event_data.CallTrace : "*dbgcore*") and
```
Comment (brokensound77) [inline rules/windows/credential_access_suspicious_lsass_access_memdump.toml:30]:
```suggestion
  winlog.event_data.CallTrace : ("*dbghelp*", "*dbgcore*") and
```

**Review by brokensound77 — APPROVED** [review]
Few small tweaks then LGTM

**Pull request merged**